bookmarklet.js

```diff
@@ -31,6 +31,16 @@ javascript:(
                 document.body.removeChild(dummy);
             };
 
+            const downloadFile = (text) => {
+                var dataStr = "data:text/json;charset=utf-8," + encodeURIComponent(text);                
```

Comment:
pls no var, only `const` or `let`

Reply:
you can change it, it was copied from stackoverflow

Reply:
just updated the code